v2/charts/azure-service-operator/values.yaml

```diff
@@ -13,6 +13,15 @@ azureClientID: ""
 # This is required when using Service Principal authentication.
 azureClientSecret: ""
 
+# [Deprecated] Managed Identity (aad-pod-identity) Authentication
+# azureManagedIdentityId is the client ID (00000000-0000-0000-0000-00000000000) of the Azure Managed Identity used to authenticate with Azure if using aad-pod-identity(deprecated) authorization.
+# azureManagedIdentityResourceId is the resource ID (/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managedIdentityName})
+# of the Azure Managed Identity used to authenticate with Azure if using aad-pod-identity authorization.
+aadPodIdentity:
+  enable: false
+  azureManagedIdentityClientId: ""
```

Comment:
I think you can just use the existing azureClientID variable. We don't need a separate one specific to aadPodIdentity.